Source/Core/Core/AchievementManager.cpp

```diff
@@ -212,6 +214,43 @@ AchievementManager::ResponseType AchievementManager::FetchGameData()
       rc_api_process_fetch_game_data_response);
 }
 
+void AchievementManager::ActivateDeactivateAchievement(AchievementId id, bool enabled,
+                                                       bool unofficial, bool encore)
```

Comment:
As an aside... this kind of logic is logic that I would expect to be in the library, not in the emulator -- surely the library has a better idea what combination of flags result in an active or inactive achievement than any user if it would. Kinda weird to me that the emulator has to implement it?

Reply:
None of these flags have any bearing on what's going on in the library - "encore mode" doesn't even appear on the site at all, it's basically a bonus feature that some emulators have but isn't required in all of them.